src/lib/components/02-components/branding/branding.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from 'next/link';
+
+export type BrandingProps = {
+  href?: string;
+  src: string;
+  alt: string;
+  width?: string;
+  height?: string;
+};
+
+export default function Branding({
+  href = '/',
+  alt,
+  src,
+  width,
+  height,
+  ...props
+}: BrandingProps) {
+  return (
+    <Link href={href} {...props}>
+      <img
+        src={src}
+        alt={`${alt}`}
```

Comment:
No need to use a template literal here.

```suggestion
        alt={alt}
```